src/core/lv2/Lv2SubPluginFeatures.cpp

```diff
@@ -67,54 +69,128 @@ Lv2SubPluginFeatures::Lv2SubPluginFeatures(Plugin::Type type) :
 
 
 
-void Lv2SubPluginFeatures::fillDescriptionWidget(QWidget *parent,
-													const Key *k) const
+void addHbox(QWidget* parent, const char* left, QString right)
 {
-	const LilvPlugin *plug = getPlugin(*k);
-
-	auto label = new QLabel(parent);
-	label->setText(QWidget::tr("Name: ") + pluginName(plug));
+	if(right.isEmpty()) { return; }
 
-	auto label2 = new QLabel(parent);
-	label2->setText(QWidget::tr("URI: ") +
-		lilv_node_as_uri(lilv_plugin_get_uri(plug)));
-
-	auto maker = new QWidget(parent);
-	auto l = new QHBoxLayout(maker);
+	auto container = new QWidget(parent);
+	auto l = new QHBoxLayout(container);
 	l->setContentsMargins(0, 0, 0, 0);
 	l->setSpacing(0);
 
-	auto maker_label = new QLabel(maker);
-	maker_label->setText(QWidget::tr("Maker: "));
-	maker_label->setAlignment(Qt::AlignTop);
+	auto leftLabel = new QLabel(container);
+	leftLabel->setText(QWidget::tr(left));
```

Comment:
I believe you are missing quotes, or is it this way?

Reply:
Quotes where exactly/for what?

Reply:
Within the tr()

Reply:
You mean `tr("left")`? I want it to really use the text which had been passed to the function, not the constant string `"left"`.

Reply:
Ahh, why did i leave such a dum review?

Reply:
All good, thanks for trying!